source/ios/tools/IOSFeed/IOSFeed.cs

```diff
@@ -0,0 +1,140 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Azure.Storage;
+using Microsoft.Azure.Storage.Blob;
+
+namespace IOSFeedNS
+{
+    class IOSFeed
+    {
+        public static void Main()
+        {
+            // Run the examples asynchronously, wait for the results before proceeding
+            ProcessAsync().GetAwaiter().GetResult();
+        }
+
+        private static async Task ProcessAsync()
+        {
+            string storageConnectionString;
+
+            var connectionStringPath = "source/ios/tools/IOSFeed/ConnectString.txt";
+            using (StreamReader sr = File.OpenText(connectionStringPath))
+            {
+                storageConnectionString = sr.ReadToEnd();
+            }
+
+            Console.WriteLine("key string: " + storageConnectionString);
+            // Check whether the connection string can be parsed.
+            CloudStorageAccount storageAccount;
+            if (CloudStorageAccount.TryParse(storageConnectionString, out storageAccount))
+            {
+                // If the connection string is valid, proceed with operations against Blob
+                // storage here.
+                // ADD OTHER OPERATIONS HERE
+                Console.WriteLine("success!");
```

Comment:
> Console.WriteLine("success!"); [](start = 15, length = 31)

Remove console.writeline and instead use the System.Diagnostics.Debug class if you need to. Also for the commented todo's go ahead prefix them with a TODO: pls! thx